HITDAD/js/plugins/SPFCore.js

```diff
@@ -246,11 +246,7 @@ function SPF_ParseNote(event) {
     var enemyEvents = [];
     events.forEach(function(event) {
 
-      var eventJSON = SPF_ParseNote(event);
-
-      if (!SPF_isEmpty(eventJSON) &&
-          eventJSON.npcType == SPF_NPCS.SECURITY_NPC) {
-
+        if (event._npcType === 1) { // <npc_type:1> is normal guard
```

Comment:
We should change the SPF_NPCS in SPFCore.js to account for this:

```
var SPF_NPCS = {
  NORMAL_GUARD: 1
};
```
So that when you reference this, you can use `event.npcType = SPF_NPCS.NORMAL_GUARD`. This lets us document with code rather then comments.